src/dashboards/Compact/index.tsx

```diff
@@ -53,14 +53,33 @@ const EnturDashboard = ({ history }: Props): JSX.Element => {
     const localStorageLayout = getFromLocalStorage(history.location.key)
     const extraCols = anyBikeRentalStations ? 1 : 0
 
+    // Limit column count to n if there are not enough space
+    function maxCols(n: number): number {
+        return numberOfStopPlaces + extraCols < n
```

Comment:
Vil ikke dette bli det samme som `Math.min(numberOfStopPlaces + extraCols, n)` eller leser jeg det feil?

Reply:
Lurer og på om det er et bedre navn for funksjonen, som f.eks `limitToMax(columns: number)` ikke sikkert det er det beste, men noe som beskriver effekten/intensjonen bedre.

Reply:
Godt forslag! Forandrer på det nå.